omod/src/main/webapp/portlets/patientOverview.jsp

```diff
@@ -34,8 +35,145 @@
 			</td>
 		</tr>
 	</table>
+	<!--Copied from OpenMRS core 1.9.x
+	See https://github.com/openmrs/openmrs-core/blob/1.9.x/webapp/src/main/webapp/WEB-INF/view/portlets/patientOverview.jsp#L36
```

Comment:
Same as above for using tags

Reply:
[Fixed](https://github.com/openmrs/openmrs-module-legacyui/pull/128/commits/9dd92c8a5560377daedb2e231c639e315fa177b8)